src/components/DocumentationLayout/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+import MainLayout, { LayoutComponent, LayoutModifiers } from '../MainLayout'
+import ThemeDocumentationLayout from 'gatsby-theme-iterative-docs/src/components/Documentation/Layout'
+
+const Layout: LayoutComponent = ({ children, ...restProps }) => {
+  const {
+    location: { pathname }
+  } = restProps
+
+  return (
+    <MainLayout
+      {...restProps}
+      modifiers={[LayoutModifiers.Wide, LayoutModifiers.Collapsed]}
+    >
+      <ThemeDocumentationLayout currentPath={pathname}>
+        {children}
+      </ThemeDocumentationLayout>
+    </MainLayout>
+  )
+}
+
+export default Layout
```

Comment:
In the latest iteration, I noticed the inner part of `DocumentationLayout` is pretty necessary to make the whole thing work, but thankfully only `MainLayout` pulls in a bunch of website dependencies so it can be easily broken off here.